src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -197,8 +198,15 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
         </Button>
       );
     }
-    // should never happen but we can't return undefined without some refactoring
-    return <span />;
+
+    return (
+      <RequestIntegrationButton
+        organization={organization}
+        name={provider.name}
+        slug={provider.slug}
+        type="integration"
```

Comment:
@mgaeta I thought I commented this already but I can't find it now. I think we can make a `renderRequestIntegrationButton` in the abstract base class so if we need to update props, we only have to do it in one place instead of 3.

Reply:
This is a really great idea. Everything is a lot simpler